app/views/subscriptions/_actions.html.erb

```diff
@@ -1,3 +1,5 @@
+<% css_class = local_assigns[:dots_menu] ? 'dropdown-item' : '' %>
```

Comment:
Is it correct to say that `dots_menu` is true for `#show` actions? If so, we can drop this local variable and use something like:
```
<% css_class = action_name == 'show' ? 'dropdown-item' : '' %>
```

this way, we won't have to pass `dots_menu` in the views anymore.

Reply:
Yes, but that's not the reason for this. 

There subscribe/unsubscribe link is rendered in two places in `#show`. It needs to have a `dropdown-item` class in the dots-menu but it can't use that class in the subscriptions sections.

I used this local variable approach to render the partial in two slightly different ways on the same view.

Reply:
In that case, I think this should be good 👍